authorization-for-rag/vercel-ai-next-js/src/lib/tools/context-docs.ts

```diff
@@ -29,13 +30,15 @@ export const getContextDocumentsTool = tool({
       buildQuery: (doc: DocumentWithScore) => ({
         user: `user:${user?.email}`,
         object: `doc:${doc.documentId}`,
-        relation: 'viewer',
+        relation: 'can_view',
```

Comment:
this model it seems we could prefer to use `can_view` (which covers `owner` and `viewer`) use case:

<img width="499" height="681" alt="Screenshot 2025-10-14 at 5 46 18 PM" src="https://github.com/user-attachments/assets/471183ea-2139-45e7-86ac-65f8872beb90" />